src/client/pages/Checkout/CheckoutDetails/components/YourPlan/YourPlan.tsx

```diff
@@ -40,20 +40,13 @@ const HorizontalDivider = styled(Divider)`
   margin: 0.5rem 0;
 `
 
-export const YourPlan = () => {
-  const priceData = useGetPriceData()
+export const YourPlan = (data: PriceData) => {
```

Comment:
Consider adding like a regular React component:
```suggestion
export const YourPlan = ({ data }: Props) => {
```